assets/scss/base/_lists.scss

```diff
@@ -1,10 +1,21 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+Unordered lists are used for representing data in a list format.
+
+Markup:
+<ul class="{{modifier_class}}">
+  <li>List item 1</li>
+  <li>List item 2</li>
+  <li>List item 3</li>
+</ul>
+
+.bullets                      - A bulleted list using the modifier `.bullets`
+.lower-letters                - A lowercase ASCII lettered list (a, b, c, etc) using the modifier `.lower-letters`
+.list--ticks                  - A ticked list item with custom tick icons using the modifier `.list--ticks`
+.list--inline                 - An inline list using the modifier `.list--inline`
+
+Styleguide Lists
 */
 
 ul, nav ul {
```

Comment:
This doesn't feel like it belongs in `lists.scss` we shouldn't have element selectors in components, it should be class selectors only. Element selectors due to the higher specificity belong in base. But please raise an issue for this linking to this problem as this is something outside of the scope of the component library commenting.